site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -137,146 +139,128 @@ extension uses a button to change the background color.
-      "128": "images/get_started128.png"
+      "16": "/images/get_started16.png",
+      "32": "/images/get_started32.png",
+      "48": "/images/get_started48.png",
+      "128": "/images/get_started128.png"
     }
-  },
-  "manifest_version": 3
+  }
 }
 ```
 
 Extensions also display images on the extension management page, the permissions warning, and
 favicon. These images are designated in the manifest under [`icons`][19].
 
-```json
+```json/18-23
 {
   "name": "Getting Started Example",
-  "version": "1.0",
   "description": "Build an Extension!",
-  "permissions": ["storage"],
+  "version": "1.0",
+  "manifest_version": 3,
   "background": {
-    "scripts": ["background.js"],
-    "persistent": false
+    "service_worker": "background.js"
   },
-  "page_action": {
+  "permissions": ["storage"],
+  "action": {
     "default_popup": "popup.html",
     "default_icon": {
-      "16": "images/get_started16.png",
-      "32": "images/get_started32.png",
-      "48": "images/get_started48.png",
-      "128": "images/get_started128.png"
+      "16": "/images/get_started16.png",
+      "32": "/images/get_started32.png",
+      "48": "/images/get_started48.png",
+      "128": "/images/get_started128.png"
     }
   },
   "icons": {
-    "16": "images/get_started16.png",
-    "32": "images/get_started32.png",
-    "48": "images/get_started48.png",
-    "128": "images/get_started128.png"
-  },
-  "manifest_version": 3
+    "16": "/images/get_started16.png",
+    "32": "/images/get_started32.png",
+    "48": "/images/get_started48.png",
+    "128": "/images/get_started128.png"
+  }
 }
 ```
 
-If the extension is reloaded at this stage, it will include a grey-scale icon, but will not contain
-any functionality differences. Because `page_action` is declared in the manifest, it is up to the
-extension to tell the browser when the user can interact with `popup.html`.
-
-Add declared rules to the background script with the [`declarativeContent`][20] API within the
-`runtime.onInstalled` listener event.
-
-```js
-chrome.runtime.onInstalled.addListener(function() {
-  chrome.storage.sync.set({color: '#3aa757'}, function() {
-    console.log('The color is green.');
-  });
-  chrome.declarativeContent.onPageChanged.removeRules(undefined, function() {
-    chrome.declarativeContent.onPageChanged.addRules([{
-      conditions: [new chrome.declarativeContent.PageStateMatcher({
-        pageUrl: {hostEquals: 'developer.chrome.com'},
-      })
-      ],
-          actions: [new chrome.declarativeContent.ShowPageAction()]
-    }]);
-  });
-});
-```
-
-The extension will need permission to access the [`declarativeContent`][21] API in its manifest.
-
-```json
-{
-  "name": "Getting Started Example",
-...
-  "permissions": ["declarativeContent", "storage"],
-...
-}
-```
+If the extension is reloaded at this stage, it will include a the provided icon rather than the
+default placeholder and clicking the action will open a popup with button with default colors.
```

Comment:
placeholder, and
(https://bit.ly/3a7EYcG)

Reply:
Resolved in next commit.